apps/gpc-verifier/app/hooks/useProofRequest.ts

```diff
@@ -0,0 +1,147 @@
+import { useMemo } from "react";
+import {
+  serializeGPCProofConfig,
+  gpcBindConfig,
+  GPCProofConfig,
+  PODMembershipLists
+} from "@pcd/gpc";
+import { POD_INT_MAX } from "@pcd/pod";
+import { generateSnarkMessageHash } from "@pcd/util";
+
+// https://docs.pcd.team/types/_pcd_gpc.GPCProofConfig.html
+const proofConfig: GPCProofConfig = {
```

Comment:
This is a great explanation of the power of GPCs.  I wonder if it's worth surfacing a copy of this directly in the README.  Not sure - maybe it's too long, but perhaps a prominent link to here as part of the description of the scenario?

Reply:
added a link in the "example use case", also in the "example code" section. let me know if that's good enough.